### PR TITLE
Update README.md with "Development" documentation

### PR DESCRIPTION
**Pull request by tomasohCHOM**
Created documentation for developing with the `discord_app` library. Clarified that they need to do Port Forwarding with ngrok by copying the URL generated from running the command `ngrok http [PREFERRED_PORT]`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,39 @@ inject type information into your interaction handlers.
 >
 > The `discord_app` library is currently only known to be available in Deno.
 
+### Development
+
+When developing with the `discord_app` library, make sure to include all of the 
+necessary environmental variables from the Discord application into its designated 
+`.env` file.
```
Comment (EthanThatOneKid) [inline README.md:26]: It may be helpful to the reader to see a list of which secrets belong in the `.env` file.

Comment (tomasohCHOM) [reply README.md]: Added necessary environmental variables in!

File: README.md [hunk 2]
```diff
@@ -19,6 +19,45 @@ inject type information into your interaction handlers.
...
+**Terminal 1:**
+
+```bash
+deno task start
+```
+
+**Terminal 2:**
+
+```bash
+ngrok http [YOUR_PREFERRED_PORT]
```
Comment (EthanThatOneKid) [inline README.md:45]:
```suggestion
ngrok http [YOUR_PORT]
```

File: README.md [hunk 3]
```diff
@@ -19,6 +19,45 @@ inject type information into your interaction handlers.
...
+### Development
+
+When developing with the `discord_app` library, make sure to include all of the 
+necessary environmental variables from the Discord application into its designated 
+`.env` file. Namely, those variables are:
+
+```
+DISCORD_APPLICATION_ID
+DISCORD_PUBLIC_KEY
+DISCORD_TOKEN
```
Comment (EthanThatOneKid) [inline README.md:31]:
```suggestion
DISCORD_APPLICATION_ID=""
DISCORD_PUBLIC_KEY=""
DISCORD_TOKEN=""
```

**Review by EthanThatOneKid — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tomasohCHOM) [reply README.md:45]: Nice!

File: README.md (hunk 3 above)
Comment (tomasohCHOM) [reply README.md:31]: Makes sense!

**Review by tomasohCHOM — COMMENTED** [review]
LGTM!

**Review by EthanThatOneKid — APPROVED** [review]
LGTM! Thank you for updating the documentation, @tomasohCHOM :godmode:

**Pull request merged**